.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,154 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  
+  cmake_test:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: sudo ./ci/setup_cmake.sh ./ci/setup_ci_environment.sh
+    - name: run tests
+      run: ./ci/do_ci.sh cmake.test
+
+  plugin_test:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: sudo ./ci/setup_cmake.sh ./ci/setup_ci_environment.sh
+    - name: run tests
+      run: ./ci/do_ci.sh cmake.test_example_plugin
+  
+  gcc_48_test:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: sudo ./ci/setup_ci_environment.sh ./ci/install_bazelisk.sh ./ci/install_gcc48.sh
+    - name: run tests
+      run: ./ci/do_ci.sh bazel.legacy.test
+      
+  bazel_test:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: sudo ./ci/setup_cmake.sh ./ci/setup_ci_environment.sh ./ci/install_bazelisk.sh
+    - name: run tests
+      run: ./ci/do_ci.sh bazel.test
+
+  bazel_noexcept:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: sudo ./ci/setup_cmake.sh ./ci/setup_ci_environment.sh ./ci/install_bazelisk.sh
+    - name: run tests
+      run: ./ci/do_ci.sh bazel.noexcept
+
+  bazel_asan:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: sudo ./ci/setup_cmake.sh ./ci/setup_ci_environment.sh ./ci/install_bazelisk.sh
+    - name: run tests
+      run: ./ci/do_ci.sh bazel.asan
+
+  bazel_tsan:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup
+      run: sudo ./ci/setup_cmake.sh ./ci/setup_ci_environment.sh ./ci/install_bazelisk.sh
+    - name: run tests
+      run: ./ci/do_ci.sh bazel.tsan
+
+  benchmark:
```

Comment:
On CircleCI, we're able to see the benchmark results, e.g.: https://3864-193156399-gh.circle-artifacts.com/0/benchmark/api/test/trace/span_id_benchmark_result.txt

How can I see the results on GitHub Actions? I couldn't find them on https://github.com/open-telemetry/opentelemetry-cpp/pull/103/checks?check_run_id=758560225

Reply:
@g-easy This might fit: https://github.com/marketplace/actions/continuous-benchmark ? :)

Reply:
or publish the benchmark text files directly: https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts

Reply:
You can try this:

```
  benchmark:
    name: Run benchmarks
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: setup
       run: |
        sudo ./ci/setup_cmake.sh
        sudo ./ci/setup_ci_environment.sh
        sudo ./ci/install_bazelisk.sh
    - name: run tests
       run: |
        env BENCHMARK_DIR=/benchmark
        ./ci/do_ci.sh benchmark
    - name: Upload benchmark results
       uses: actions/upload-artifact@v2
       with:
        name: benchmark_reports
        path: /benchmark
```

Reply:
Thanks! The benchmark results should now be downloadable and viewable after the job runs on the GitHub Actions tab.